backend/src/interactions/updateUnconfirmedContribution/UnconfirmedContributionAdmin.role.ts

```diff
@@ -37,20 +61,40 @@ export class UnconfirmedContributionAdminRole extends AbstractUnconfirmedContrib
       !role.hasRight(RIGHTS.MODERATOR_UPDATE_CONTRIBUTION_MEMO) &&
       this.self.moderatorId === null
     ) {
-      throw new LogError('An admin is not allowed to update an user contribution')
+      throw new LogError("The Moderator hasn't the right MODERATOR_UPDATE_CONTRIBUTION_MEMO")
     }
     return this
   }
 
   protected async validate(clientTimezoneOffset: number): Promise<void> {
     await super.validate(clientTimezoneOffset)
+
+    const newResubmissionDate = this.updateData.resubmissionAt
+      ? new Date(this.updateData.resubmissionAt)
+      : null
+
+    const resubmissionNotChanged =
+      this.self.resubmissionAt !== null &&
+      newResubmissionDate !== null &&
+      +this.self.resubmissionAt === +newResubmissionDate
+
     // creation date is currently not changeable
     if (
-      this.self.memo === this.updateData.memo &&
-      this.self.amount === this.updatedAmount &&
-      this.self.contributionDate.getTime() === new Date(this.updatedCreationDate).getTime()
+      !this.isContributionChanging() &&
+      ((this.self.resubmissionAt === null && newResubmissionDate === null) ||
+        resubmissionNotChanged)
     ) {
       throw new LogError("the contribution wasn't changed at all")
     }
```

Comment:
the leading comment, the if-expression and the exception-message is quite confusing.
- comment says: somthing is not changeable
- if-expression checks: if somethings is changed in contribution and resubmission-Attributes are still unchanged
- exception-msg says: contribution is unchanged at all

so if something has changed, which is not allowed, than the exception-message should describe this more expressive.